languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/disambiguation.xml

```diff
@@ -237,6 +237,17 @@
       </pattern>
       <disambig action="remove" postag="V.*"/>
     </rule>
+    <rule>
+      <pattern>
+        <token postag="V.+" postag_regexp="yes"/>
+        <token min='1' max='2' postag="[DP].+" postag_regexp="yes"/>
+        <token postag="Z0[FM]P0" postag_regexp="yes"/>
+        <marker>
+          <token postag="VMIP1S0" postag_regexp="no"/>
```

Comment:
If this rule is only for "como", you could write explicitly "como" there. It will be safer.

Reply:
@jaumeortola 

I am not sure if it only works with “como”.

It also works with “logo”.

I based myself on the rule I was working on.

I was expecting to look at the night diff to see the results and later fix it in case there are false positives.

Reply:
Anyway, something is missing. If the word inside the `<marker>` is only verb (or verb and noun), all POS tags will be removed. I would add some other conditions: specific words (como, logo), or other POS tags (CS, PT, RG...)

Reply:
@jaumeortola 

I have done it hours ago.

Let me refresh the review.